bokehjs/src/less/_mixins.less

```
@@ -66,3 +66,17 @@
   background-position: center center;
   background-repeat: no-repeat;
 }
+
+.bk-disabled(@disabled-opacity, @disabled-background-color, @disabled-color, @icon-color-disabled) {
+ &[disabled], &.bk-disabled {
```

This indentation is wrong.


```suggestion
  &[disabled], &.bk-disabled {
```